### PR TITLE
[Modal] - Checks if wrapper is clicked before closing

### PR DESCRIPTION
**Pull request by kierangillen**
- Only closes modal if the wrapper is clicked

### CODE REVIEW

File: packages/palette/src/elements/Modal/Modal.tsx [hunk 1]
```diff
@@ -178,8 +179,11 @@ export const Modal: SFC<ModalProps> = ({
   return (
     <>
       {renderModal && (
-        <ModalOuterWrapper show={show} onClick={() => handleWrapperClick()}>
-          <ModalWrapper>
+        <ModalOuterWrapper show={show}>
+          <ModalWrapper
+            ref={wrapperRef}
+            onClick={event => handleWrapperClick(event)}
```
Comment (damassi) [inline packages/palette/src/elements/Modal/Modal.tsx]: `onClick={handleWrapperClick}`

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit aba2954e299a11929c54f9e295c5003d5e550310

https://deploy-preview-581--artsy-palette.netlify.com

**Comment by erikdstock** [comment]
#mergeongreen

**Comment by artsyit** [comment]
:rocket: PR was released in v4.17.2 :rocket:

**Pull request merged**